frameworks/PHP/amp/server.php

```diff
@@ -12,42 +12,46 @@
 use Amp\Http\Server\Server;
 use Amp\Promise;
 use Amp\Success;
-use Psr\Log\NullLogger;
+use Monolog\Logger;
+
+define('DB_HOST', gethostbyname('tfb-database'));
 
 Amp\Loop::run(function () {
     $sockets = yield [
-        Cluster::listen("0.0.0.0:8080"),
-        Cluster::listen("[::]:8080"),
+        Cluster::listen('0.0.0.0:8080')
```

Comment:
This should probably pass a customized `BindContext` and increase the default backlog size.

https://github.com/amphp/socket/blob/abc18f1786363ceee2555ffd7fe87e4eead2ce24/src/BindContext.php#L12

Reply:
Yes, 128 is very low for a server.
But I don't know enough Amp.
Please could you help with that?

Reply:
Sure. All you need to do is create an instance of `(new Amp\Socket\BindContext)->withBacklog($size)` as second  parameter.

Reply:
Added socket bind context with `backlog(102400)` as the stream server use 2 sockets.
And with `reuseport` and `tcpnodelay`.

But I miss and option to use `keepalive` in the `BindContest class`.

Reply:
@joanhey I stumbled upon that yesterday, too, so I just opened https://github.com/amphp/socket/issues/67.

Reply:
Without `keepalive`, never will be fast, as each request need to open a new tcp connection.

Reply:
Oh, we're talking about different keep alives. HTTP/1.1 keep alive / connection reuse is automatically enabled.

The thing I just opened is about Tcp keep alive and only affects idle connections.